docs/learning/WriteAConnector.md

```diff
@@ -146,7 +141,7 @@ An iTwin Connector provides a workflow to easily synchronize information from va
 
 #### Briefcases
 
-[Briefcases](./glossary#briefcase) are the local copies of iModel that users can acquire to work with the iModel. A Connector will download a briefcase locally using the BridgeRunner and change their copy of iModel. Once all the work is done, the results are then pushed into the iModel. Please see the section on [Execution sequence](#execution-sequence) on the different steps involved.
+[Briefcases](./glossary#briefcase) are the local copies of iModel that users can acquire to work with the iModel. A Connector will download a briefcase locally using the ConnectorRunner and change their copy of iModel. Once all the work is done, the results are then pushed into the iModel. Please see the section on [Execution sequence](#execution-sequence) on the different steps involved.
```

Comment:
_I know this doesn't pertain to this PR, but might as well clean this up_
This is a bit confusing due to inconsistent pluralization.

```suggestion
[Briefcases](./glossary#briefcase) are the local copies of iModels users can acquire. A Connector will download a briefcase locally using the ConnectorRunner and change the local copy of their iModel. Once all the work is done, the results are then pushed into the iModel. Please see the section on [Execution sequence](#execution-sequence) on the different steps involved.
```

Reply:
@ben-polinsky thanks for pointing this out - I agree that it was confusing.  I tried to make it more efficient by simply saying briefcase is a local copy and then refer to the briefcase w/o saying local or imodel.  I think it is straightforward now.